### ISSUE TITLE
constraint "ltss_pkey" of relation "ltss" does not exist causes migration to fail to v.2.1.0

### ISSUE THREAD

**Issue by antoonhuiskens** [freol35241/ltss#111]
Ok. Just had another go at upgrading:
I'm getting this:
```
Error during connection setup: (psycopg2.errors.UndefinedObject) constraint "ltss_pkey" of relation "ltss" does not exist [SQL: ALTER TABLE ltss DROP CONSTRAINT ltss_pkey CASCADE, ADD PRIMARY KEY(time,entity_id);] (Background on this error at: https://sqlalche.me/e/20/f405) (retrying in 3 seconds)
```

Checking on compression:
```
# SELECT hypertable_name,chunk_name,is_compressed FROM timescaledb_information.chunks LIMIT 1;
 hypertable_name |    chunk_name    | is_compressed
-----------------+------------------+---------------
 ltss            | _hyper_1_1_chunk | f
```

I'm not sure why the ltss_constraint doesn't exist.

I'd say that adding IF EXISTS to the ALTER TABLE statement would fix this.
(1 row)

Doesn't look to be related to role permissions either:
```
# \z ltss
                                 Access privileges
 Schema | Name | Type  |      Access privileges      | Column privileges | Policies
--------+------+-------+-----------------------------+-------------------+----------
 public | ltss | table | ltssadmin=arwdDxt/ltssadmin+|                   |
        |      |       | grafanareader=r/ltssadmin   |                   |
```

**Comment by antoonhuiskens** [comment]
In hindsight, the ltss_pkey was likely DROP'ed in the first pass.

**Comment by balk77** [comment]
I think I had the same issue. I had a constraint called `ltss_pkey_merge` . Renamed it to `ltss_merge` and that fixed the issue. I cannot trace back because it was fixed 👍

**Comment by freol35241** [comment]
Closed with #112
